test/socket.cpp

```diff
@@ -291,6 +433,75 @@ BOOST_AUTO_TEST_CASE(accept_receive_send___connect_send_receive)
     ios.run();
 }
 
+BOOST_AUTO_TEST_CASE(accept___close)
```

Comment:
Still don't like the tripple underscores. Prefer explicit name like `accept_then_close`.